dcap/ql/src/lib.rs

```diff
@@ -8,5 +8,24 @@ extern crate alloc;
 mod quote3;
 mod quote_enclave;
 
+use mc_sgx_dcap_types::Quote3Error;
 pub use quote3::TryFromReport;
-pub use quote_enclave::{load_policy, set_path, QeTargetInfo};
+pub use quote_enclave::{load_policy, PathInitializer, QeTargetInfo};
+
+/// Errors interacting with quote library functions
+#[derive(Clone, Debug, displaydoc::Display, Eq, Hash, PartialEq, PartialOrd, Ord)]
+#[non_exhaustive]
+pub enum Error {
+    /// Paths have already been initialized
+    PathsInitialized,
+    /// Error from SGX quoting library function: {0}
+    Sgx(Quote3Error),
```

Comment:
I've added a task to #145 to further expand this error. I think it's best for #164 to get settled before investing too much here, also this PR already got bigger than I wanted :(.